core/src/main/java/org/geysermc/geyser/item/type/Item.java

```diff
@@ -228,7 +228,23 @@ public void translateNbtToJava(@NonNull NbtMap bedrockTag, @NonNull DataComponen
         // TODO streamline Enchantment process
         Enchantment.JavaEnchantment enchantment = Enchantment.JavaEnchantment.of(enchantId);
         if (enchantment == Enchantment.JavaEnchantment.SWEEPING_EDGE) {
-            addSweeping(session, builder, level);
+            String sweepingTranslation = MinecraftLocale.getLocaleString("enchantment.minecraft.sweeping", session.locale());
+            addJavaOnlyEnchantment(session, builder, sweepingTranslation, level);
+            return null;
+        }
+        if (enchantment == Enchantment.JavaEnchantment.DENSITY) {
+            String densityTranslation = MinecraftLocale.getLocaleString("enchantment.minecraft.density", session.locale());
+            addJavaOnlyEnchantment(session, builder, densityTranslation, level);
+            return null;
+        }
+        if (enchantment == Enchantment.JavaEnchantment.BREACH) {
+            String breachTranslation = MinecraftLocale.getLocaleString("enchantment.minecraft.breach", session.locale());
+            addJavaOnlyEnchantment(session, builder, breachTranslation, level);
+            return null;
+        }
+        if (enchantment == Enchantment.JavaEnchantment.WIND_BURST) {
+            String windBurstTranslation = MinecraftLocale.getLocaleString("enchantment.minecraft.wind_burst", session.locale());
+            addJavaOnlyEnchantment(session, builder, windBurstTranslation, level);
             return null;
         }
```

Comment:
seeing how all of these essentially do the same thing - maybe it's worth adding a map field of enchantment -> translation (string), and then checking if that map contains this java only enchantment. That would get rid of some code duplication :)

for example - we hardcore some values with a map in our recipe translator:
https://github.com/GeyserMC/Geyser/blob/e697eb3ae38bbda0b1b4803ea03a73067881dec2/core/src/main/java/org/geysermc/geyser/translator/protocol/java/JavaUpdateRecipesTranslator.java#L98

Reply:
Yep, you're right. Just new to the repo and wasn't too familiar with the code style around here. Changed! :)